descricão básica dos tipos

mateustranquilino opened the pull request.

mateustranquilino committed:
descricão básica dos tipos

issilva5 commented on Haskell/Model/Medico.hs:
```diff
@@ -0,0 +1,8 @@
+    id :: Int,
+    nome :: String,
+    crm :: String,
+    -- idUbs :: Int
```
remover linha comentada

issilva5 commented on Haskell/Model/UBS.hs:
```diff
@@ -0,0 +1,6 @@
+data UBS = UBS {
+    id :: Int,
+    nome :: String,
+    -- medicos :: [Int],
```
Tá faltando a relação médico-UBS, achei que tu tinha comentado lá em médico porque ia por aqui. Mas aqui tá comentado tbm. Acho que fica mais fácil, deixar o código lá, e retirar aqui.

issilva5 reviewed (changes requested):
Adiciona a definição do módulo em cada uma dos arquivos de tipo. Tu pode usar aquela definição que higor mostrou pra quando está dentro de uma pasta. ex.: Model.Receita.

mateustranquilino committed:
Definição de módulos para os arquivos de tipo.
Alteração dos atributos "data" para "dia",
pois "data" é palavra reservada de Haskell .

issilva5 committed:
atualizando o gitignore...
